samples/bootloader/CMakeLists.txt

```diff
@@ -6,7 +6,29 @@
 
 cmake_minimum_required(VERSION 3.8.2)
 
+
 include($ENV{ZEPHYR_BASE}/cmake/app/boilerplate.cmake NO_POLICY_SCOPE)
 project(bootloader)
 
-target_sources(app PRIVATE src/main.c)
+# This is for allowing the bootloader sample as a separate image in a
+# multi-image build context. It also allows it to be built as a sample
```

Comment:
IMO we don't need to state that it is possible to build it as a sample as before, this is assumed to not be broken.

Reply:
So we should only state the first sentence and skip the rest.

Reply:
Okei will fix it.